Makefile

```diff
@@ -0,0 +1,43 @@
+
+# Destroys your kind cluster completely
+kinddown:
+	bazel run //dev/kind:delete
+
+# Creates a new kind cluster for you
+kindup:
+	bazel run //dev/kind:start
+
+# Lists all bazel targets, so you can grep for what you need
+bazel-targets:
+	bazel query 'attr(visibility, "//visibility:public", ...)' && \
+	bazel query 'attr(visibility, "", ...)'
+
+# Installs/updates KubeCF (on whatever cluster your kubectl points to)
+apply:
+	bazel run //dev/kubecf:apply
+
+# Installs the CF operator (TODO: this should be done with bazel)
+operatorup:
+	k create namespace cf-operator && \
+	bazel run //dev/cf_operator:apply
+
+# Cleans everything up, but doesn't destroy your cluster
+# (TODO: this should be done with bazel)
+clean:
```

Comment:
It implies `k` is installed on the host, which would break our "user contract", no?

Reply:
sure, i missed that